plugins/modules/user.py

```diff
@@ -138,6 +139,8 @@ def modify_user(module):
             if attr == 'load_module':
                 load_module_opts = "-R %s " % val
             else:
+                if val in [True, False, "True", "False", "Yes", "No", "Always", "Never"]:
```

Comment:
Does this cover all cases even if  letters  of mixed cases/all uppercases are passed in as strings by the user like TrUE , fALse, yEs , FALSE etc.,